clang/test/SemaSYCL/num_simd_work_items_device.cpp

```diff
@@ -42,45 +42,45 @@ struct FuncObj {
 #ifdef TRIGGER_ERROR
 // If the declaration has an [[intel::reqd_work_group_size]] or
 // [[cl::reqd_work_group_size]] attribute, tests that check if
-// the work group size attribute argument (the first argument)
+// the work group size attribute argument (the last argument)
 // can be evenly divided by the num_simd_work_items attribute.
 struct TRIFuncObjBad1 {
   [[intel::num_simd_work_items(3)]]        // expected-error{{'num_simd_work_items' attribute must evenly divide the work-group size for the 'reqd_work_group_size' attribute}}
-  [[intel::reqd_work_group_size(5, 3, 3)]] // expected-note{{conflicting attribute is here}}
+  [[intel::reqd_work_group_size(5, 3, 5)]] // expected-note{{conflicting attribute is here}}
```

Comment:
Could you please change the examples to something like (3, 3, 5) so we know which value the complaint is about?  With (5, 3, 5) it is ambiguous as to which '5' the '3' does not divide.   This comment applies to many of the cases in this test.

Reply:
Done